### PR TITLE
Support for Nested structs (#133)

### PR DESCRIPTION
**Pull request by mannprerak2**
Co-authored-by: J-P Nurmi <jpnurmi@gmail.com>

### CODE REVIEW

**Comment by mannprerak2** [comment]
@dcharkes It's giving some errors it didn't before because it uses the latest dev version for the test. 
Can we ignore the tests since they will be fixed in the upcoming commits?

**Comment by mannprerak2** [comment]
Hey, @jpnurmi  could you please sign the CLA by commenting `@googlebot I consent.`.

We were moving some commits from `dart-dev` to `master` in individual PRs. we might need this one more time after this.

**Comment by jpnurmi** [comment]
@googlebot I consent.

File: .github/workflows/test-package.yml [hunk 1]
```diff
@@ -3,9 +3,9 @@ name: Dart CI
 on:
   # Run on PRs and pushes to the default branch.
   push:
-    branches: [ master ]
+    branches: [ dart-dev, master ]
```
Comment (dcharkes) [inline .github/workflows/test-package.yml:6]: Is this on purpose? Should we only have `[ master ]` there until we start using the stable branch and then this should be `[ stable, master]`?

**Review by dcharkes — APPROVED** [review]
Already reviewed in https://github.com/dart-lang/ffigen/pull/133.

File: .github/workflows/test-package.yml (hunk 1 above)
Comment (mannprerak2) [reply .github/workflows/test-package.yml:6]: Okay sure

**Comment by mannprerak2** [comment]
@dcharkes for some reason (I suspect the CLA error before) the CI is stuck at queued state (see https://github.com/dart-lang/ffigen/actions)

File: .github/workflows/test-package.yml (hunk 1 above)
Comment (mannprerak2) [reply .github/workflows/test-package.yml]: Updated to only master for now.

**Comment by dcharkes** [comment]
No CI shows up at all on this PR. Just the CLA bot. Where are:

* Dart CI / analyze (beta) (pull_request)
* Dart CI / test (pull_request)

**Comment by dcharkes** [comment]
I've deleted the workflow run, let's see if the subsequent ones work now.

**Comment by mannprerak2** [comment]
@dcharkes can you delete all the queued up workflows? I can then try and push an empty commit to see if that works.

**Comment by dcharkes** [comment]
> @dcharkes can you delete all the queued up workflows? I can then try and push an empty commit to see if that works.

"Failed to cancel workflow" - that doesn't work apparently.

@athomas any idea what's going on here?

**Comment by mannprerak2** [comment]
Github actions is down currently https://www.githubstatus.com/

**Comment by mannprerak2** [comment]
@dcharkes all working now 👍🏻

**Comment by dcharkes** [comment]
Sweet! 👍

**Pull request merged**